helmfile/charts/starboard/starboard-operator-psp-rbac/templates/starboard-operator-psp-rbac.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: 00-starboard-operator
+  annotations:
```

Comment:
Why is this starting with `00-`? is to be picked over the default one?

Reply:
Yes, it is because starboard operator does not set a proper security context on its scans, so by spec they can get the `restricted` PSP which limits them so they can't run.

Reply:
Ah, ok. Then maybe add a comment about that at the start of this file, just so we remember.

Reply:
Added a comment.